pkg/importer/spanner/sysl.arrai

```diff
@@ -0,0 +1,138 @@
+# size returns the size of an attribute
+let size = \length
+    cond {
+        length = 'MAX': '',
+        //seq.has_prefix('0x', length): '(' ++ //seq.trim_prefix('0x', length) ++ ')',
+        length > 0: $`(${length})`,
+        _: '',
```

Comment:
Nit: This line is redundant. If no cond branch matches, the whole thing evals to `{}`.

Same for all other such lines below.

Reply:
ok